src/plugins/intel_gpu/include/intel_gpu/op/dynamic_quantize.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+
+namespace ov {
+namespace intel_gpu {
+namespace op {
+
+/// \brief Operator performing Dynamic Quantize
+class DynamicQuantize : public ov::op::Op {
+public:
+    OPENVINO_OP("DynamicQuantize", "gpu_opset");
+
+    DynamicQuantize() = default;
+    /// \brief Constructs an DynamicQuantize operation.
+    ///
+    /// \param data Input tensor with data
+    DynamicQuantize(const Output<Node>& data, int64_t group_size);
```

Comment:
Then is it necessary to propagate this change to properties too?

Reply:
I don't think so. Even if you change value of property to the vector, it will not make it generic as different operation (or even instances of same op) can have different ranks, so single property will never be aligned with that. My vision is that we know the context where we insert DynamicQuantize op (currently - before compressed FC), so we can understand what is expected tensor rank and what are the requirements in terms of quantization for current op and how to apply this group_size parameter properly

Reply:
Could you elaborate more about why you think such direction is better? I guess one advantage would be the explicit expression of group_size parameter to which axis it will be applied. But if non-innermost-axis will be always 1, I'm not sure whether it is worth the complexity..